packages/theme/src/theme/_fonts.scss

```
@@ -1,127 +1,104 @@
+* {
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+}
+
```

body selection should be enough
```suggestion
body {
	// sass-lint:disable-block no-vendor-prefixes
	-webkit-font-smoothing: antialiased;
	-moz-osx-font-smoothing: grayscale;
}

```